cookbooks/aws-parallelcluster-install/attributes/default_centos7.rb

```diff
@@ -9,7 +9,7 @@
                                          libical-devel postgresql-devel postgresql-server sendmail libxml2-devel libglvnd-devel
                                          mdadm python python-pip libssh2-devel libgcrypt-devel libevent-devel glibc-static bind-utils
                                          iproute NetworkManager-config-routing-rules python3 python3-pip iptables libcurl-devel yum-plugin-versionlock
-                                         coreutils moreutils sssd sssd-tools sssd-ldap curl environment-modules)
+                                         coreutils moreutils sssd sssd-tools sssd-ldap curl environment-modules rpm-build)
```

Comment:
Previously we opted for adding the package installation directly to the resource which depends on it.

It better surfaces the dependencies and, as a side effect, avoids depending on `install_packages` which is costly.

I would follow the same path here. Does it make sense?

Reply:
Moved in the corresponding resource